test/sql/test_bitmap_functions/T/test_bitmap_functions

```diff
@@ -69,3 +69,58 @@ select bitmap_to_string(bitmap_subset_in_range(v1, 3, 2)) from test_bitmap_table
 select bitmap_to_string(bitmap_subset_in_range(v1, 2, 3)) from test_bitmap_table1 order by k1;
 select bitmap_to_string(bitmap_subset_in_range(v1, -2, 3)) from test_bitmap_table1 order by k1;
 
+-- name: test_bitmap_binary
+CREATE TABLE `t1` (
+  `c1` int(11) NULL COMMENT "",
+  `c2` bitmap BITMAP_UNION NULL COMMENT ""
+) ENGINE=OLAP
+AGGREGATE KEY(`c1`)
+DISTRIBUTED BY HASH(`c1`) BUCKETS 1
+PROPERTIES ("replication_num" = "1");
+
+-- empty bitmap
+insert into t1 values (1, bitmap_empty());
+select c1, hex(bitmap_to_binary(c2)) from t1;
+select c1, bitmap_count(bitmap_from_binary(bitmap_to_binary(c2))) from t1;
+
+-- single 32bit bitmap
+truncate table t1;
+insert into t1 values (1, to_bitmap(1));
+select c1, hex(bitmap_to_binary(c2)) from t1;
+select c1, bitmap_to_string(bitmap_from_binary(bitmap_to_binary(c2))) from t1;
+
+-- single 64bit bitmap
+truncate table t1;
+insert into t1 values (1, to_bitmap(17179869184));
+select c1, hex(bitmap_to_binary(c2)) from t1;
+select c1, bitmap_to_string(bitmap_from_binary(bitmap_to_binary(c2))) from t1;
+
+-- set bitmap
+truncate table t1;
+insert into t1 select 1, bitmap_agg(generate_series) from table(generate_series(1, 5));
+select c1, hex(bitmap_to_binary(c2)) from t1;
+select c1, bitmap_to_string(bitmap_from_binary(bitmap_to_binary(c2))) from t1;
+
+-- RoaringBitmap32
+truncate table t1;
+insert into t1 select 1, bitmap_agg(generate_series) from table(generate_series(1, 40));
+select c1, hex(bitmap_to_binary(c2)) from t1;
+select c1, bitmap_to_string(bitmap_from_binary(bitmap_to_binary(c2))) from t1;
+
+-- RoaringBitmap64
+truncate table t1;
+insert into t1 select 1, bitmap_agg(generate_series) from table(generate_series(1, 20));
+insert into t1 select 1, bitmap_agg(generate_series) from table(generate_series(17179869184, 17179869284));
+select c1, hex(bitmap_to_binary(c2)) from t1;
+select c1, bitmap_to_string(bitmap_from_binary(bitmap_to_binary(c2))) from t1;
```

Comment:
need to add some tests when inputs is null.

Reply:
Fixed